pytest_trio/plugin.py

```diff
@@ -39,6 +41,12 @@ def pytest_addoption(parser):
         type="bool",
         default=False,
     )
+    parser.addini(
+        "trio_run",
+        "what runner should pytest-trio use? [trio, qtrio]",
+        type="linelist",
```

Comment:
This only needs to be a string, not a list of strings, right? I think you can just not pass a type and it'll be interpreted as a single string.

Reply:
https://github.com/pytest-dev/pytest/pull/7872